frontend/packages/shared/src/components/AppUserProfileMenu/AppUserProfileMenu.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { type ReactElement } from 'react';
+import { type Repository, type User } from 'app-shared/types/Repository';
+import { useTranslation } from 'react-i18next';
+import { useUserNameAndOrg } from './hooks/useUserNameAndOrg';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import {
+  useMediaQuery,
+  StudioProfileMenu,
+  StudioAvatar,
+  type StudioProfileMenuProps,
+} from '@studio/components';
+import { MEDIA_QUERY_MAX_WIDTH } from 'app-shared/constants';
+
+export type AppUserProfileMenuProps = {
+  user: User;
+  repository: Repository;
+  color: StudioProfileMenuProps['color']; // TODO - See if we can do this with "StudioProfileMenuItem" too
+  profileMenuItems: StudioProfileMenuProps['profileMenuItems'];
```

Comment:
Is it OK to export the `StudioProfileMenuItem` type from `@studio/components`? Or should we have it like I've done above? This goes for a lot of files. 
@framitdavid